test/parallel/test-repl-underscore.js

```diff
@@ -178,7 +178,7 @@ function testError() {
 
       // The sync error, with individual property echoes
       /Error: ENOENT: no such file or directory, scandir '.*nonexistent.*'/,
-      /fs\.readdirSync/,
+      /Object\.readdirSync/,
```

Comment:
The `module.exports` pattern actually makes the stack frame harder to read...sign.